README.md

```diff
@@ -1,15 +1,133 @@
-## Scaffolding the client with Vite and Tailwind
+# ethTO
 
+Cross-chain NFT renting marketplace.
+
+## Table of Contents
+
+- [Scaffolding](#scaffolding)
+- [Contributing](#contributing)
+  - [Types](#types)
+  - [Branches](#branches)
+  - [Commits](#commits)
+  - [Pull Requests](#pull-requests)
+  - [Merging Into Main](#merging-into-main)
+
+## Scaffolding
+
+### Scaffolding the client with Vite and Tailwind
+
+```shell
 cd /client
 npm install,
 npm run dev
+```
 
-## Scaffolding the protocol with Hardhat
+### Scaffolding the protocol with Hardhat
 
+```shell
 cd /protocol,
 npm install
+```
 
-## Scaffolding the oracles
+### Scaffolding the oracles
 
+```shell
 cd /oracles,
 npm install
+```
+
+## Contributing
+
+Code changes can fall into the types from the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
```

Comment:
Just a proposal, this convention wouldn't be enforced by something like `commitlint` unless we configure it